Package:     squidguard
Subject:     add ipv6 support
Author:      Joachim Wiedorn <joodebian at joonet.de>
Origin:      upstream, http://www2.squidguard.org/Bugs/show_bug.cgi?id=54
Last-Update: 2015-11-25

Now that squid 3.0 announced ip v6 support, squidGuard should be able
to handle this as well.
---

diff -urN s14/src/sgDiv.c s15/src/sgDiv.c
--- s14/src/sgDiv.c	2015-11-25 00:05:04.906975023 +0100
+++ s15/src/sgDiv.c	2015-11-25 00:16:46.554488769 +0100
@@ -211,7 +211,7 @@
     *(s->surl + (e - p)) = '\0';
   }
   //strcpy(s->surl,s->domain);
-  if(strspn(s->domain,".0123456789") == strlen(s->domain))
+  if(strspn(s->domain,":.0123456789") == strlen(s->domain))
     s->dot = 1;
   if(d != NULL)
     strcat(s->surl,d);
diff -urN s14/src/sgDiv.c.in s15/src/sgDiv.c.in
--- s14/src/sgDiv.c.in	2015-11-25 00:05:04.906975023 +0100
+++ s15/src/sgDiv.c.in	2015-11-25 00:16:46.554488769 +0100
@@ -211,7 +211,7 @@
     *(s->surl + (e - p)) = '\0';
   }
   //strcpy(s->surl,s->domain);
-  if(strspn(s->domain,".0123456789") == strlen(s->domain))
+  if(strspn(s->domain,":.0123456789") == strlen(s->domain))
     s->dot = 1;
   if(d != NULL)
     strcat(s->surl,d);
